crates/oxc_traverse/src/ast_operations/gather_node_parts.rs

```diff
@@ -22,8 +22,8 @@ pub fn get_var_name_from_node<'a, N: GatherNodeParts<'a>>(node: &N) -> String {
 
     if name.is_empty() {
         name = "ref".to_string();
-    } else {
-        name.truncate(20);
+    } else if name.len() > 20 {
```

Comment:
Should probably have a comment here explaining why this needs to be done this way, if possible :)